event/parsing/metrics.go

```diff
@@ -17,9 +17,10 @@ const (
 // Measures tracks the various event-related metrics.
 type Measures struct {
 	fx.In
-	MetadataFields        metrics.Counter   `name:"metadata_fields"`
-	BootTimeHistogram     metrics.Histogram `name:"boot_time_duration"`
-	UnparsableEventsCount metrics.Counter   `name:"unparsable_events_count"`
+	MetadataFields         metrics.Counter   `name:"metadata_fields"`
+	BootTimeHistogram      metrics.Histogram `name:"boot_time_duration"`
+	TotalBootTimeHistogram metrics.Histogram `name:"total_boot_time_duration"`
```

Comment:
This may be a confusing name if later an event before `reboot-pending` or after `fully-manageable` is added.  I don't currently have any suggestions for something better though... 🤔